### DISCUSSION TITLE
Could be good to have sorting feature in milestone page

### DISCUSSION THREAD

**Discussion by VaishnavJois** [EddieHubCommunity/LinkFree#8924] [Ideas]
Could be good to have sorting feature in milestone page. Currently it just adds in the order a new one is added. Maybe a simple chronological order option based on dates milestones were achieved?!

**Comment by eddiejaoude** [comment]
I think sorting is not needed but having it in descending order is a good idea 👍  - please check this has not been done yet and if not raise an issue
